src/pages/signin/PasswordForm.js

```diff
@@ -87,6 +88,14 @@ class PasswordForm extends React.Component {
         Session.resetPassword();
     }
 
+    /**
+    * Clears any previously entered 2FA Code and signin Onyx data
+    */
+    clearSignInData() {
+        this.setState({twoFactorAuthCode: ''});
+        Session.clearSignInData();
+    }
```

Comment:
I know it is not exactly part of the original issue, but I think we should also clear the `formError` from the state. This fixes this case:

- Enter email of existing account
- Enter bad password (you will get an error)
- Go back
- Enter email of existing account
- The error is still there

<img width="492" alt="Screen Shot 2022-09-01 at 12 50 00 PM" src="https://user-images.githubusercontent.com/87341702/187897355-68ef42aa-eb50-4747-aeb6-515e1fddba8b.png">

Reply:
Definitely!